extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/SonosAudioSink.java

```diff
@@ -92,9 +93,9 @@ public void process(AudioStream audioStream) throws UnsupportedAudioFormatExcept
                         logger.warn("Sonos speaker '{}' is not initialized - status is {}", handler.getThing().getUID(),
                                 handler.getThing().getStatus());
                     } else if (AudioFormat.WAV.isCompatible(format)) {
-                        handler.playNotificationSoundURI(new StringType(url + ".wav"));
+                        handler.playNotificationSoundURI(new StringType(url + FileAudioStream.WAV_EXTENSION));
                     } else if (AudioFormat.MP3.isCompatible(format)) {
-                        handler.playNotificationSoundURI(new StringType(url + ".mp3"));
+                        handler.playNotificationSoundURI(new StringType(url + FileAudioStream.MP3_EXTENSION));
```

Comment:
I really don't know, but isn't it a little bit unexpected that an "extension" constant contains the "filename-extension-separator" (the dot)?